README.md

```diff
@@ -635,7 +635,7 @@ on your own.
 * The JIT compiler produces an unsafe program: memory access are not tested at
   runtime (yet). Use with caution.
 
-* Contrary to the interpreter, if a division by 0 is attempted, the JIT program
+* If a division by 0 is attempted by a JIT-ted program, the program
```

Comment:
The remark was more about the `panic` than the “contrary” :). Sorry, I didn't make that clear. Something like

```
Contrary to the interpreter, if a division by 0 is attempted, the JIT program
returns `0xffffffffffffffff` and exits cleanly (no `Error` returned). This is
because the author has not found how to return something more explicit
from the generated assembly so far.
```

Thanks for the commas.